### PR TITLE
Add canMakePayments method

### PR DESCRIPTION
**Pull request by beylmk**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,39 +1,6093 @@
 {
   "name": "cordova-plugin-purchases",
   "version": "2.1.1",
-  "lockfileVersion": 1,
```
Comment (aboedo) [inline package-lock.json:4]: can't express how much I regret having this file in version control 😂

File: plugin.xml [hunk 2]
```diff
@@ -33,8 +34,9 @@
                 <source url="https://github.com/CocoaPods/Specs.git"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="PurchasesHybridCommon" spec="1.6.2"/>
+                <pod name="PurchasesHybridCommon" spec="1.7.1"/>
             </pods>
         </podspec>
     </platform>
+    <params name="ANDROIDXENABLED" value="true" />
```
Comment (aboedo) [inline plugin.xml:41]: ooohhh nice, I hadn't seen this before and I'd always enable it manually per-app

File: src/android/PurchasesPlugin.java [hunk 3]
```diff
@@ -7,13 +7,14 @@
...
+import com.revenuecat.purchases.hybridcommon.CommonKt;
+import com.revenuecat.purchases.hybridcommon.ErrorContainer;
+import com.revenuecat.purchases.hybridcommon.OnResult;
+import com.revenuecat.purchases.hybridcommon.OnResultList;
+import com.revenuecat.purchases.hybridcommon.OnResultAny;
 import com.revenuecat.purchases.common.PlatformInfo;
-import com.revenuecat.purchases.common.SubscriberAttributesKt;
-import com.revenuecat.purchases.common.mappers.PurchaserInfoMapperKt;
+import com.revenuecat.purchases.hybridcommon.SubscriberAttributesKt;
+import com.revenuecat.purchases.hybridcommon.mappers.PurchaserInfoMapperKt;
```
Comment (aboedo) [inline src/android/PurchasesPlugin.java:17]: 🎉

**Review by aboedo — APPROVED** [review]

File: tests/plugin.test.ts [hunk 4]
```diff
@@ -58,4 +58,77 @@ describe("Purchases", () => {
...
+          expect(execFn).toHaveBeenCalledWith(
+            expect.any(Function),
+            expect.any(Function),
+            "PurchasesPlugin",
+            "canMakePayments",
+            expected
+          );
+        });
+      });
+  });
```
Comment (vegaro) [inline tests/plugin.test.ts:133]: yay tests!

**Review by vegaro — APPROVED** [review]
good job!

**Pull request merged**